### DISCUSSION TITLE
How can i zoom in with a Camera2dBundle?

### DISCUSSION THREAD

**Discussion by DasLixou** [bevyengine/bevy#5904] [Q&A] [answered]
Hi! I wanna zoom with my Camera2dBundle that i apply like this: `commands.spawn_bundle(Camera2dBundle::default());`.i cant make a Camera2dBundle and just change the transform with a scale because the other values should be generated via the new_with_far function.

**Comment by brandon-reinhart** [comment, marked as answer]
In a system, get the OrthographicProjection associated with the Camera2d and set the scale there:

```
fn zoom(mut query: Query<&mut OrthographicProjection, With<Camera2d>>) {
   let mut projection = query.single_mut();
   projection.scale = 5.0;
}
```

Spawning the bundle is the same as spawning and then inserting a bundle. Inserting a bundle is inserting several components simultaneously. When you use a bundle, it's worth looking up the payload of components that are added. For Camera2dBundle, that's here:

https://docs.rs/bevy/latest/bevy/core_pipeline/core_2d/struct.Camera2dBundle.html

You can see in the list it includes an OrthographicProjection which you can manipulate.
